catalyst/contrib/dl/runner/wandb.py

```diff
@@ -16,7 +16,9 @@ class WandbRunner(Runner):
     Runner wrapper with wandb integration hooks.
     """
     @staticmethod
-    def _log_metrics(metrics: Dict, mode: str, suffix: str = ""):
+    def _log_metrics(
+        metrics: Dict, mode: str, suffix: str = "", epoch: int = None
+    ):
```

Comment:
`_log_metrics` is used for both batch and epoch metrics, so `epoch: int = None` looks a bit confusing in this way

Reply:
I can change parameter name of the `epoch` to `step` and we can somehow (I don't know how and possibly this would be a hack) obtain `batch_idx` we can try to put it there OR there is a solution without this parameter:
Changing `wandb.log(metrics)` to `wandb.log(metrics, commit=False)`

We can also provide parameter `commit` for `_log_metrics` which would be default to `True`. Should we try this one solution?

Reply:
I've changed this one to `commit=False`. That's actually better solution.